### PR TITLE
feat: implement UI for logging out

### PR DESCRIPTION
**Pull request by gorjan5sk**
Implemented Log out in Preferences -> Account: 
- Logging out of current session  
- Logging out of all other sessions
- Manage sessions

### CODE REVIEW

File: app/assets/javascripts/preferences/panes/account/LogOutView.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        any time.
+      </Text>
+      <div className="min-h-6" />
+      <div className="flex flex-row items-center w-full">
+        <Button
+          type="normal"
+          label="Cancel"
+          onClick={() => {
+            closeDialog();
+          }}
```
Comment (vardan-arm) [inline app/assets/javascripts/preferences/panes/account/LogOutView.tsx:36]:
```suggestion
          onClick={closeDialog}
```

File: app/assets/javascripts/preferences/panes/account/LogOutView.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    <ConfirmationDialog title={<Title>End your session?</Title>}>
+      <Text>This will delete all your local items and preferences.</Text>
+      <div className="min-h-6" />
+      <div className="flex flex-row items-center w-full">
+        <Button
+          type="normal"
+          label="Cancel"
+          onClick={() => {
+            closeDialog();
+          }}
```
Comment (vardan-arm) [inline app/assets/javascripts/preferences/panes/account/LogOutView.tsx:80]:
```suggestion
          onClick={closeDialog}
```

**Review by vardan-arm — APPROVED** [review]

File: app/assets/javascripts/preferences/panes/account/LogOutView.tsx (hunk 1 above)
Comment (gorjan5sk) [reply app/assets/javascripts/preferences/panes/account/LogOutView.tsx]: This has been changed to a different trigger function that accepts an ardgument.

File: app/assets/javascripts/components/OtherSessionsLogout.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+const ConfirmOtherSessionsLogout = observer(
+  ({ application, appState }: Props) => {
+
+    const cancelRef = useRef<HTMLButtonElement>();
+    function closeDialog() {
+      appState.accountMenu.setOtherSessionsLogout(false);
+    }
+
+    return <AlertDialog onDismiss={closeDialog} leastDestructiveRef={cancelRef}>
```
Comment (antsgar) [inline app/assets/javascripts/components/OtherSessionsLogout.tsx:32]: Could we please add a newline here after return so it's easier to read?

**Review by antsgar — APPROVED** [review]

File: app/assets/javascripts/components/OtherSessionsLogout.tsx (hunk 3 above)
Comment (gorjan5sk) [reply app/assets/javascripts/components/OtherSessionsLogout.tsx]: Done.

**Pull request merged**